nifi-framework-bundle/nifi-framework/nifi-web/nifi-web-api/src/main/java/org/apache/nifi/web/api/dto/DtoFactory.java

```diff
@@ -4116,7 +4117,16 @@ private List<ThreadDumpDTO> createThreadDumpDtos(final ProcessorNode procNode) {
        return threadDumps;
    }
 
-   /**
+    // Pattern to match a parameter reference i.e. "#{anything}"
+    private static final Pattern PARAMETER_REFERENCE = Pattern.compile("^#\\{.*}$");
```

Comment:
This regular-expression-based approach should be replaced with parsing capabilities from the `nifi-parameter` module.

Reply:
I understand and will change it to use nifi-parameter method(s). Thanks for pointing that out.